client/app/redux/actions/MenuItemInformationActions.js

```diff
@@ -0,0 +1,35 @@
+import * as types from './types';
+import { get } from '../../lib/api-utility';
+
+export function getMenuItemInformation(itemID) {
+
+    const request = () => {
+        return { type: types.GET_MENU_ITEM_INFORMATION_REQUEST }
+    }
+
+    const success = (menuItem) => {
+        return {
+            type: types.GET_ALL_DINING_HALLS_INFORMATION_SUCCESS,
```

Comment:
Should this be `GET_MENU_ITEM_INFORMATION_SUCCESS`?

Reply:
Thank you :)